timelock/program/src/utils.rs

```diff
@@ -15,8 +15,41 @@ use solana_program::{
     system_instruction::create_account,
     sysvar::rent::Rent,
 };
+
+use std::str::FromStr;
+
 use spl_token::state::Account;
 
+/// Pubkey used as None value placeholder
+pub fn none_pubkey() -> Pubkey {
+    Pubkey::from_str("11111111111111111111111111111111").unwrap()
```

Comment:
I think the better way to do this is to store it as a 33 byte array. The first byte is 1 or 0, 1 indicates the next 32 bytes are a readable public key, a 0 indicates it is not and you should just return None. Then you don't need a "None" pubkey at all. You don't even care what's in that part of the slice. If someone passes in None and you decide to serialize None, you just flip the first bit to 0. That's how borsh does it.

Reply:
Yes, this is what I proposed as an alternative solution in the other PR